src/tests/readytorun/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -31,13 +33,17 @@ public static int TestEntryPoint()
 
         Console.WriteLine("\nHello World from Jitted Methods Counting Test!");
 
+        // Get the total amount of Jitted Methods.
         long jits = JitInfo.GetCompiledMethodCount(false);
```

Comment:
```suggestion
        long jits = JitInfo.GetCompiledMethodCount(currentThread: false);
```
This is an edit that would make this code easier to follow. It is not obvious by just looking at the code what the `false` argument means.